src/main/java/org/eclipse/xpanse/terraform/boot/models/validation/TerraformValidationResult.java

```diff
@@ -16,7 +18,13 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class TerraformValidationResult {
 
-    private String terraformVersion;
+    @NotNull
+    @Schema(description = "Defines if the Terraform scripts is valid.")
     private boolean valid;
+
+    @Schema(description = "Terraform with version which executed the command.")
+    private String terraformWithVersion;
```

Comment:
terraformVersionUsed
The version of the Terraform binary used to execute scripts